inst/extdata/energy/A44.HouseholdSize.csv

```diff
@@ -0,0 +1,5 @@
+# File: A44.HouseholdSize.csv
+# Title: Assumed household size for translating the Odyssee floorspace per house
```

Comment:
Added a comment:
Odyssee is a database for energy efficiency indicators in Europe (http://www.odyssee-indicators.org). Referenced (no. 15) in PNNL-21073 China's Building Energy Use: A Long-Term Perspective based on a Detailed Assessment.